src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,61 @@
+.stopwatch {
+  position: relative;
+  height: $main-size;
+  width: $main-size;
+
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__seconds {
+    position: absolute;
+    height: $second-height;
+    width: 1.5vmin;
+    top: 2vmin;
+    left: 50%;
+
+    transform: translateX(-50%);
+    transform-origin: bottom center;
+    animation: move 60s linear 0s infinite;
+    animation-fill-mode: backwards;
```

Comment:
It looks like the 'move' keyframes are not defined within this file or the provided files. Ensure that the keyframes for the 'move' animation are defined somewhere within the SCSS files. If they are missing, the animation will not work as expected.